test/testUtils/listenersManager.ts

```diff
@@ -0,0 +1,41 @@
+import { expect } from "vitest";
+
+/**
+ * Whether or not it is currently the first time running this manager.
+ */
+let firstTime = true;
+
+/**
+ * The list of listeners that were present during the first time this manager is run.
+ * These initial listeners are needed throughout the entire test suite, so we never remove them.
+ */
+const initialListeners: unknown[] = [];
+
+/**
+ * The current listener that is only needed for the current test file.
+ * We plan to delete it during the next test file, when it is no longer needed.
+ */
+let currentListener;
```

Comment:
```suggestion
let currentListener: any;
```
Should this be `any` or `unknown`?